analysis/test/golden_test.sh

```diff
@@ -58,7 +58,7 @@ check_gitlogparser_repo_scan() {
   echo " -- expect GitLogParser repo-scan subcommand to produce valid cc.json file"
   ACTUAL_GITLOG_JSON_REPO_SCAN="${INSTALL_DIR}/actual_gitlogparser_repo_scan.cc.json"
   returnCode="0"
-  timeout 60s "${CCSH}" gitlogparser "repo-scan" --repo-path "$(dirname "$(dirname "$PWD")")" -o "${ACTUAL_GITLOG_JSON_REPO_SCAN}" -nc || returnCode="$?"
+  timeout 120s "${CCSH}" gitlogparser "repo-scan" --repo-path "$(dirname "$(dirname "$PWD")")" -o "${ACTUAL_GITLOG_JSON_REPO_SCAN}" -nc || returnCode="$?"
```

Comment:
Is it really needed to double up the time due to your changes?

Reply:
Not required due to my changes, but rather due to my system. I was not able to fully run the integration test, because the command took so long. 
The change does not make the tests slower (only if it would fail, because it needs to wait longer until timeout).
 If you want, I can revert this change, and use a local version for testing?

Reply:
Personally, I would not push those changes that are ment only for my local system.

Reply:
I definitly agree, but it makes sense for this command, because it is a 2-in-1, so it should be able to run for longer than the other one.

Reply:
Ah ok, then the merge of the two gitlog parser commands does increase the needed time. I see. I would recommend to use the lowest possible/useful time.